client/my-sites/marketplace/pages/marketplace-product-details/index.tsx

```diff
@@ -43,16 +43,20 @@ import { isAtomicSiteWithoutBusinessPlan } from 'calypso/blocks/eligibility-warn
 import { requestEligibility } from 'calypso/state/automated-transfer/actions';
 import Notice from 'calypso/components/notice';
 import { eligibilityHolds } from 'calypso/state/automated-transfer/constants';
+import { getDefaultPluginInProduct } from 'calypso/my-sites/marketplace/marketplace-product-definitions';
 
 interface MarketplacePluginDetailsInterface {
-	marketplacePluginSlug: keyof PluginProductMappingInterface;
```

Comment:
Since this line was removed, `PluginProductMappingInterface` is no longer in use.